Document/0x05a-Platform-Overview.md

```diff
@@ -48,60 +105,103 @@ For example, Android 7.0 (API level 24) defines the following system users:
+Example: `android.permission.RECORD_AUDIO`
+- *Signature*: This permission is granted only if the requesting app has been signed with the same certificate as the app that declared the permission. If the signature matches, the permission is automatically granted.
+Example: `android.permission.ACCESS_MOCK_LOCATION`
+- *SystemOrSignature*: This permission is granted only to apps embedded in the system image or signed with the same certificate that the app that declared the permission was signed with.
+Example: `android.permission.ACCESS_DOWNLOAD_MANAGER`
 
-##### Adiantum
+Requesting Permissions
 
-AES is used on most modern Android devices for storage encryption. Actually, AES has become such a widely used algorithm that the most recent processor implementations have a dedicated set of instructions to provide hardware accelerated encryption and decryption operations, such as ARMv8 with its Cryptography Extensions or x86 with AES-NI extension.
-However, not all devices are capable of using AES for storage encryption in a timely fashion. Especially low-end devices running Android Go. These devices usually use low-end processors, such as the ARM Cortex-A7 which don't have hardware accelerated AES.
+Apps can request permissions for the protection levels Normal, Dangerous, and Signature by including `<uses-permission />` tags into their manifest.
+The example below shows an AndroidManifest.xml sample requesting permission to read SMS messages:
 
-Adiantum is a cipher construction designed by Paul Crowley and Eric Biggers at Google to fill the gap for that set of devices which are not able to run AES at least at 50 MiB/s. Adiantum relies only on additions, rotations and XORs; these operations are natively supported on all processors. Therefore, the low-end processors can encrypt 4 times faster and decrypt 5 times faster than they would if they were using AES.
+```xml
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.permissions.sample" ...>
 
-Adiantum is a composition of other ciphers:
+    <uses-permission android:name="android.permission.RECEIVE_SMS" />
+    <application>...</application>
+</manifest>
+```
 
-- NH: A hashing function.
-- Poly1305: A message authentication code (MAC).
-- XChaCha12: A stream cipher.
-- AES-256: A single invocation of AES.
+Declaring Permissions
 
-Adiantum is a new cipher but it is secure, as long as ChaCha12 and AES-256 are considered secure. Its designers didn't create any new cryptographic primitive, instead they relied on other well-known and thoroughly studied primitives to create a new performant algorithm.
+Apps can expose features and content to other apps installed on the system. To restrict access to its own components, it can either use any of Android’s [predefined permissions](https://developer.android.com/reference/android/Manifest.permission.html "predefined permissions") or define its own. A new permission is declared with the `<permission>` element.

```

Comment:
> "predefined permissions"

Perhaps use the page's title ("Manifest.permission") as title. Or just remove the link's title, if it is the same as the link's text.

Reply:
All links must have a title according to our style guide.